.gitignore

```diff
@@ -2,3 +2,4 @@ package-lock.json
 *egg-info
```

Comment:
Could egg and package lock be removed as a part of removing the npm and pip processes?

Reply:
yep.